incompatibility/apiset.go

```diff
@@ -14,28 +14,26 @@
 
 package incompatibility
 
+// IntermediateReport counts the number of incompatibility occurrences
+type IntermediateReport struct {
```

Comment:
Should this type be public?

Reply:
needed for return rest of countincompatibilities used in analysis_test